clang/lib/Driver/ToolChains/Arch/AArch64.cpp

```diff
@@ -81,6 +81,15 @@ static bool DecodeAArch64Features(const Driver &D, StringRef text,
     else
       return false;
 
+    // +jsconv and +complxnum implies +neon and +fp-armv8
```

Comment:
I don't think we need these implications. Features dependencies are already set in llvm/lib/Target/AArch64/AArch64.td

`def FeatureComplxNum : SubtargetFeature<
    "complxnum", "HasComplxNum", "true",
    "Enable v8.3-A Floating-point complex number support (FEAT_FCMA)",
    [FeatureNEON]>;`
   
`def FeatureJS : SubtargetFeature<
    "jsconv", "HasJS", "true",
    "Enable v8.3-A JavaScript FP conversion instructions (FEAT_JSCVT)",
    [FeatureFPARMv8]>;`

Reply:
Indeed. The change in the driver has been removed.